mercury.spec

```diff
@@ -26,6 +26,7 @@ Group:    Development/Libraries
 License:  Argonne National Laboratory, Department of Energy License
 URL:      http://mercury-hpc.github.io/documentation/
 Source0:  https://github.com/mercury-hpc/mercury/archive/v%{dl_version}.tar.gz
+Patch0:   https://github.com/daos-stack/mercury/na_ucx_local_addr.patch
```

Comment:
You don't need the URL here since the patch is local, you should only need a URL when the patch is upstreamed into mercury master.

Reply:
Indeed.  This URL is a 404 even.  This should be fixed before this can land.